ansible_ai_connect/users/auth.py

```diff
@@ -67,10 +67,24 @@ def extra_data(self, user, uid, response, details=None, *args, **kwargs):
         return data
 
     def user_has_valid_license(self, access_token):
-        url = f"{settings.AAP_API_URL}/v2/config/"
+        url = self.get_config_endpoint(settings.AAP_API_URL)
         data = self.get_json(url, headers={"Authorization": f"bearer {access_token}"})
         return not data["license_info"]["date_expired"] if "license_info" in data else False
 
+    def get_me_endpoint(self, api_url):
+        """Creates me link to the AAP API depending on the Auth platform"""
+
+        # AAP Controller has /api at the end for API link, AAP Gateway doesn't
+        url = api_url.rstrip("/")
+        return f"{url}/v2/me/" if url.endswith("/api") else f"{url}/api/gateway/v1/me/"
```

Comment:
Maybe extract `v1` and `v2` and turn them into constants?

Reply:
v1 and v2 are from different APIs one if for v2 is version of API of controller and v1 is version of Gateway (new API).

So next in line is Gateway v2, but no clarity if other part of the link will change as well.